terraform/eks/main.tf

```diff
@@ -160,9 +161,10 @@ resource "kubernetes_service_account" "aoc-role" {
 }
 
 resource "kubernetes_service_account" "aoc-fargate-role" {
+  count = 1
```

Comment:
Shouldn't this also only be created on fargate deployments? It seems fargate specific.

Reply:
Should've refreshed.